test/LazyVectorIndexTest.cc

```diff
@@ -1097,55 +1098,52 @@ TEST_CASE_METHOD(VectorSearchTest, "TestIndexUpdaterIndexOutOfBounds", "[VectorS
     
     array<int, 2> indices { -1, 1 };
     for (auto i : indices) {
-        {
-            // This is in line with FLArray, returns null when index is out-of-bound.
-            ExpectingExceptions x {};
-            auto value = CBLIndexUpdater_Value(updater, i);
-            CHECK(!value);
-        }
+        // Note: These two functions cannot be tested as they will abort when index is out-of-bound.
```

Comment:
We made this undefined behavior now, so really there is no reason to test this.  It's not legal to do and that means the behavior is "whatever".

Reply:
Agreed. Will remove it in later commit.